packages/requisitions/src/RequestRequisition/DetailView/Toolbar/ToolbarActions.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {
+  Autocomplete,
+  InputWithLabelRow,
+  useTranslation,
+  useConfirmationModal,
+} from '@openmsupply-client/common';
+import { useRequestFields, useIsRequestDisabled } from '../../api';
+
+const months = [1, 2, 3, 4, 5, 6];
+
+export const ToolbarActions = () => {
+  const t = useTranslation('replenishment');
+  const isDisabled = useIsRequestDisabled();
+  const { minMonthsOfStock, update } = useRequestFields('minMonthsOfStock');
+  const getConfirmation = useConfirmationModal({
+    title: t('heading.are-you-sure'),
+    message: t('messages.changing-min-mos'),
+  });
+
+  return (
+    <InputWithLabelRow
+      labelWidth="150px"
+      labelProps={{ sx: { fontSize: 12, fontWeight: 500 } }}
+      label={t('label.min-months-of-stock')}
+      Input={
+        <Autocomplete
+          disabled={isDisabled}
+          clearIcon={null}
+          isOptionEqualToValue={(a, b) => a.value === b.value}
+          value={{
+            label: `${minMonthsOfStock} months`,
+            value: minMonthsOfStock,
+          }}
+          width="150px"
+          options={months.map(numberOfMonths => ({
+            label: t('label.number-months', { numberOfMonths }),
+            value: numberOfMonths,
+          }))}
```

Comment:
:nit - use translation for the value display and plurals
```suggestion
          value={{
            label: t('label.number-months', { count: minMonthsOfStock }),
            value: minMonthsOfStock,
          }}
          width="150px"
          options={months.map(numberOfMonths => ({
            label: t('label.number-months', { count: numberOfMonths }),
            value: numberOfMonths,
          }))}
```

Reply:
sweet

> !